sparse/impl/KokkosSparse_spmv_struct_impl.hpp

```diff
@@ -1435,7 +1435,7 @@ static void spmv_alpha_beta_mv_struct_transpose(
     // team_size is a hardware resource thing so it might legitimately
     // be int.
     const int rows_per_thread =
-        RowsPerThread<typename AMatrix::execution_space>(NNZPerRow);
+        VectorsPerThread<typename AMatrix::execution_space>(NNZPerRow);
```

Comment:
What is VectorsPerThread?

Reply:
It is `RowsPerThread`. That has been renamed and moved to the `CompressedSparseMatrix` parent class.